src/main/java/uk/gov/hmcts/reform/sscs/service/docmosis/PdfLetterService.java

```diff
@@ -87,10 +86,8 @@ public PdfLetterService(DocmosisPdfService docmosisPdfService, DocmosisTemplates
         String templatePath = docmosisTemplatesConfig.getCoversheets().get(languagePreference)
                 .get(wrapper.getNotificationType().getId());
         if (StringUtils.isBlank(templatePath)) {
-            throw new PdfGenerationException(
-                    String.format("There is no template for notificationType %s",
-                            wrapper.getNotificationType().getId()),
-                    new RuntimeException("Invalid notification type for docmosis coversheet."));
+            log.info("There is no template for notificationType " + wrapper.getNotificationType().getId());
+            return null;
```

Comment:
Is this an error to return null?

Reply:
This was to fix a problem where it was throwing exception for letters that have no evidence coversheet so should be fine to return null as it's handled elsewhere